golismero/common.py

```diff
@@ -1149,6 +1152,36 @@ def cookie(self, cookie):
     #--------------------------------------------------------------------------
 
     @property
+    def data(self):
+        return self._data
+
+    @data.setter
+    def data(self, data):
+        if data:
+            # Parse the data argument.
+            try:
+                if isinstance(data, unicode):
+                    data = data.encode("UTF-8")
+
+                if "=" in data:
+                    # Prepare data.
+                    data = data.replace("=", ":")
```

Comment:
Also, come to think of it, POST data is not necessarily formatted this way - that depends on the ContentEncoding header. We would need to add that use case too.